tests/test_globalconstraints.py

```diff
@@ -775,6 +775,26 @@ def test_inverse(self):
         self.assertFalse(cp.Model([cp.Inverse([x,y,x],[x,y,x])]).solve())
         self.assertRaises(TypeError,cp.Inverse,[a,b],[x,y])
         self.assertRaises(TypeError,cp.Inverse,[a,b],[b,False])
+        self.assertFalse(cp.Model([cp.Inverse([x, y, x], [x, y, x, y, x])]).solve())
+
+    def test_inverseOne(self):
```

Comment:
Nice that it has tests here, but should also be included in the tests of the test_constraints file

Reply:
yep, noticed while reviewing other PR that this file was to be modified to, currently doing it